pkg/pgmodel/migrate_test.go

```diff
@@ -1610,6 +1112,50 @@ func startContainer(ctx context.Context) (testcontainers.Container, error) {
 	return container, nil
 }
 
+func startPromContainer(ctx context.Context) (testcontainers.Container, error) {
+	// Set the storage directories permissions so Prometheus can write to them.
+	if err := os.Chmod(filepath.Join("testdata", "storage"), 0777); err != nil {
+		return nil, err
+	}
+	if err := os.Chmod(filepath.Join("testdata", "storage", "wal"), 0777); err != nil {
+		return nil, err
+	}
+
+	containerPort := nat.Port("9090/tcp")
+	storagePath, err := filepath.Abs(filepath.Join("testdata", "storage"))
```

Comment:
we need a README or something to say how data in there should be generated. I see the file committed but I have no idea how you got it.